cli/command/cluster/plugin.go

```diff
@@ -95,7 +95,7 @@ func killContainer(c cli.Interface, name string, sig string) error {
 func RunContainer(c cli.Interface, img string, dockerOpts docker, args []string, env map[string]string, f func(r io.Reader)) error {
 	containerName := fmt.Sprintf("amp-cluster-plugin-%s", stringid.GenerateNonCryptoID())
 	dockerArgs := []string{
-		"run", "-t", "--rm", "--name", containerName,
```

Comment:
why not deleting implicitly the container? Unless you make sure that you remove the container in all cases it's successfully exiting or is killed, you don't want to leave the container in a stopped state.